packages/hoprd/crates/hoprd-hoprd/src/cli.rs

```diff
@@ -175,32 +175,6 @@ pub struct CliArgs {
     )]
     pub api_token: Option<String>,
 
-    #[arg(
```

Comment:
Will this break existing setups where the arguments are given? Should we just ignore them instead?

Reply:
I'd have to test that one out. Likely yes, but the "migrated" version will be introducing more of these breaking changes both in the CLI and configuration file.

Reply:
```
Error: error: found argument '--healthCheck' which wasn't expected, or isn't valid in this context
```

Indeed. But the mechanism of healthchecking fundamentally changes, so I'd remove this option altogether.

Reply:
But then even rotsee deployments will break. I suggest we keep the cli args and mark them obsolete.

Reply:
I will put it back as obsoleted.